### PR TITLE
PF-1660 - Use timer as stable and feature check methods

### PR DESCRIPTION
**Pull request by fredcido**
Timer methods have been promoted to stable and now we also have the `miro.board.canUse` method to check for feature entitlement.

### CODE REVIEW

File: examples/breakout-rooms/src/components/BreakoutManager/BreakoutManager.tsx [hunk 1]
```diff
@@ -254,9 +260,9 @@ export const BreakoutManager: React.FC = () => {
               variant="solid-danger"
               size="x-large"
             >
-              Stop session
+              Stop session{" "}
```
Comment (mettin) [inline examples/breakout-rooms/src/components/BreakoutManager/BreakoutManager.tsx:263]: *SUPER NIT*
the space should be in the template string
```ts
                ? ` (${formatDisplayTime(timer.restDuration)})`
```

**Review by mettin — APPROVED** [review]

File: examples/breakout-rooms/src/components/BreakoutManager/BreakoutManager.tsx (hunk 1 above)
Comment (fredcido) [reply examples/breakout-rooms/src/components/BreakoutManager/BreakoutManager.tsx:263]: Good catch!

**Comment by mettin** [comment]
Side note: You forgot to update the app examples on the developer portal with this example. @horeaporutiu just updated one of his examples so this one came along for the ride.

**Comment by fredcido** [comment]
> Side note: You forgot to update the app examples on the developer portal with this example. @horeaporutiu just updated one of his examples so this one came along for the ride.

@mettin and what have I missed? How do I update it?

**Pull request merged**